src/groups/group.jl

```diff
@@ -1015,6 +1066,13 @@ Base.:\(p, ::Identity{G}) where {G<:MultiplicationGroup} = inv(p)
 Base.:\(::Identity{G}, p) where {G<:MultiplicationGroup} = p
 Base.:\(e::E, ::E) where {G<:MultiplicationGroup,E<:Identity{G}} = e
 
+adjoint_action(G::MultiplicationGroup, p, X) = p * X / p
```

Comment:
I think it's better to use the `translate_diff` and `inverse_translate_diff` machinery here and below, especially because e.g. for `SpecialOrthogonal` that is implemented more efficiently.

Reply:
Which machinery do you mean? I just checked that and I don't see anything that would make implementing this using `translate_diff` and `inverse_translate_diff` more efficient.

Reply:
What I mean is to define
```suggestion
function adjoint_action(G::MultiplicationGroup, p, Xₑ)
    e = make_identity(G, p)
    Xₚ = translate_diff(G, p, e, Xₑ, LeftAction())
    Y = inverse_translate_diff(G, p, p, Xₚ, RightAction())
    return Y
end
```
This is general enough actually to work for non-multiplication groups. But let's take the case of `SO(n)`, where all tangent vectors are represented as their translation to the Lie algebra. What Julia will execute is something like
```julia
function adjoint_action(G::SpecialOrthogonal, p, Xₑ)
    e = one(p)
    Xₚ = Xₑ
    Y = p * Xₚ * p'
    return Y
end
```
The `Xₚ * p'` is much more efficient than `Xₚ / p`, which is what I'm referring to.

If you defines this generally for `AbstractGroup`, then for `TranslationGroup`, this would simplify to
```julia
function adjoint_action(G::TranslationGroup, p, Xₑ)
    e = zero(p)
    Xₚ = Xₑ
    Y = Xₚ
    return Y
end
```
so you almost don't even need to special-case for `AdditionGroup`.

Reply:
Yes, that's a good point.